Tests/PubNubTests/Networking/Routers/MessageActionsRouterTests.swift

```diff
@@ -496,6 +455,72 @@ extension MessageActionsEndpointTests {
 
     wait(for: [expectation], timeout: 1.0)
   }
+}
+
+// MARK:- MessageAction Object
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [MARK comment should be in valid format. e.g. '// MARK: ...' or '// MARK: - ...'](https://app.codacy.com/app/PubNub/swift/pullRequest?prid=4513555)